utils/nctl/sh/utils/infra.sh

```diff
@@ -360,3 +360,36 @@ function get_count_of_started_nodes()
 {
     nctl-status | grep -v 'Not started' | wc -l
 }
+
+
+#######################################
+# Returns only if the chain has reached genesis (interpreted as era>=2)
+#######################################
+function do_await_genesis_era_to_complete() {
+    local LOG_STEP=${1:-'true'}
+    local TIMEOUT=${2:-'240'}
+    local CURRENT_ERA
+
+    if [ "$LOG_STEP" = "true" ]; then
+        log_step "awaiting genesis era to complete: timeout=$TIMEOUT"
+    fi
+
+    while :
+    do
+        CURRENT_ERA=$(get_chain_era)
+        if [ "$CURRENT_ERA" -ge "2" ]
```

Comment:
Why `≥ 2`? Isn't it enough for the genesis switch block to be finalized and the network to have progressed to era 1? Or, in other words: are we gaining anything by waiting another full era? If not, this will just make the tests require longer to complete, which might not be what we want.

Reply:
Some tests were awaiting for 1, some others for 2. I picked `2` to be on the safe side (assumed, there was a reason they waited a bit longer).